app/assets/images/payment_icons/mobicred.svg

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Or this. It would be great if the .svg files could be in similar format to the files here: https://github.com/activemerchant/payment_icons/pull/11/files